lib/assertions.js

```diff
@@ -538,6 +538,21 @@ module.exports = function (expect) {
         }(subject));
     });
 
+    // Copied from https://github.com/sindresorhus/ansi-regex
+    var ansiRegex = /(?:(?:\u001b\[)|\u009b)(?:(?:[0-9]{1,3})?(?:(?:;[0-9]{0,3})*)?[A-M|f-m])|\u001b[A-M]/g;
```

Comment:
Ah, yes, now we can. We couldn't before because it was to be used in the .spec.js file, where `require` isn't available. Fixed.